### ISSUE TITLE
Impossible to train on SDXL + some others issues

### ISSUE THREAD

**Issue by AIrtistry** [bmaltais/kohya_ss#1284]
Hi,

it's impossible for me to train on sdxl, it's just wayyyy too slow. ( I use the settings from [here](https://www.youtube.com/watch?v=AY6DMBCIZ3A) )
<img width="1268" alt="Untitled" src="https://github.com/bmaltais/kohya_ss/assets/139043392/5f21a67e-b3a0-4585-b0f9-4b43d9b60d40">

Even since I updated kohya all my trainings on 1.5 seems doing bad with the same settings I used before

I uninstalled / installed kohya and cuda 11.8 but the problem is still here

Any solutions ? Thanks

**Comment by saunderez** [comment]
What sped things up for me was increasing number of data loaders (mine is set to 10, not sure if that is optimal) as well as increasing batch size and gradient accumulation significantly. 

On a 16GB 4080
Batch 1 Grad Accumulation 1 Dataloaders 1 - my dataset of 93 images was taking at least 4.5 hours. - around 3.5it/s
Batch 2 Grad Accumulation 8 Dataloaders 10 - same dataset 2.5 hours
Batch 4 Grad Accumulation 8 Dataloaders 10 - around 2 hours - around 14/its

I suspect I can get a little bit more performance with some more data loaders but it's slow going testing changes when it takes 2 hours to get a result.
